lib/lib.go

```diff
@@ -178,6 +179,22 @@ func OptCheckConfigMigrations(interactive bool) Option {
 	}
 }
 
+// OptNoBootstrap ensures the node will not attempt to bootstrap to any other nodes
+// in the network. Should only be used when in a testing environment, like testgrounds
```

Comment:
I think this works in a number of contexts, doesn't need the "tests only" note. For example: when you want to do local discovery only

Reply:
good point!